projects/plugins/social/src/class-social-shares.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Class for the Social Shares.
+ *
+ * @package automattic/jetpack-social-plugin
+ */
+
+/**
+ * Register the Jetpack Social Shares Class.
+ */
+class Social_Shares {
+	const SOCIAL_SHARES_POST_META_KEY = '_publicize_shares';
+
+	/**
+	 * Return a list of of social shares.
+	 *
+	 * @param int $post_id The Post ID.
+	 *
+	 * @return array
+	 */
+	public static function get_social_shares( $post_id = null ) {
+		if ( empty( $post_id ) ) {
+			$post    = get_post();
+			$post_id = $post->ID ?? 0;
```

Comment:
0 will be falsy, so the second if below will return `array()` for that too. I wonder if it could be just `$post_id = $post->ID ?? 0;` and have the same semantics

Reply:
That is to cope with $post potentially being null and so an empty array should e returned